source/crud/tests/v2/aggregate-merge.yml

```diff
@@ -0,0 +1,136 @@
+            - $sort: { x : 1 }
+            - $match: { _id: { $gt: 1 } }
+            - $merge: { into: &output_collection "other_test_collection" }
+    expectations:
+      -
+        command_started_event:
+          command:
+            aggregate: *collection_name
+            pipeline: *pipeline
+    outcome: &outcome
+      collection:
+        name: *output_collection
+        data:
+          - { _id: 2, x: 22 }
+          - { _id: 3, x: 33 }
+  -
+    description: "Aggregate with $merge and batch size of 0"
+    operations:
+      -
+        object: collection
+        name: aggregate
+        arguments:
+          <<: *arguments
+          batchSize: 0
+    expectations:
+      -
+        command_started_event:
+          command:
+            aggregate: *collection_name
+            pipeline: *pipeline
+            # TODO: check that cursor.batchSize does not exist once
+            # https://jira.mongodb.org/browse/SPEC-1215 has been resolved.
+    outcome: *outcome
+  -
+    description: "Aggregate with $merge and majority readConcern"
+    operations:
+      -
+        object: collection
+        name: aggregate
+        collectionOptions:
+          readConcern: { level: "majority" }
+        arguments: *arguments
+    expectations:
+      -
+        command_started_event:
+          command:
+            aggregate: *collection_name
+            pipeline: *pipeline
+            readConcern: { level: "majority" }
+    outcome: *outcome
+  -
+    description: "Aggregate with $merge and local readConcern"
+    operations:
+      -
+        object: collection
+        name: aggregate
+        collectionOptions:
+          readConcern: { level: "local" }
+        arguments: *arguments
+    expectations:
+      -
+        command_started_event:
+          command:
+            aggregate: *collection_name
+            pipeline: *pipeline
+            readConcern: { level: "local" }
+    outcome: *outcome
+  -
+    description: "Aggregate with $merge and available readConcern"
+    operations:
+      -
+        object: collection
+        name: aggregate
+        collectionOptions:
+          readConcern: { level: "available" }
+        arguments: *arguments
+    expectations:
+      -
+        command_started_event:
+          command:
+            aggregate: *collection_name
+            pipeline: *pipeline
+            readConcern: { level: "available" }
+    outcome: *outcome
+  -
+    description: "Aggregate with $merge and linearizable readConcern"
+    operations:
+      -
+        object: collection
+        name: aggregate
+        collectionOptions:
+          readConcern: { level: "linearizable" }
+        arguments: *arguments
+        error: true
+    expectations:
+      -
+        command_started_event:
+          command:
+            aggregate: *collection_name
+            pipeline: *pipeline
+            readConcern: { level: "linearizable" }
+  -
+    description: "Aggregate with $merge and invalid readConcern"
```

Comment:
How strongly do you feel we need this test?  In Java the level is an enumerated value so this isn't even possible and we'd have to skip the test anyway.  More general argument against this: we're just testing the server here, not the driver.

Reply:
If anything, this is testing https://github.com/mongodb/specifications/blob/master/source/read-write-concern/read-write-concern.rst#unknown-levels-and-additional-options-for-string-based-readconcerns but I agree that there is little value to this test. Although Java may restrict the level to an enum value, isn't it possible to still specify a completely unsupported string using the [URI option](https://github.com/mongodb/specifications/blob/master/source/read-write-concern/read-write-concern.rst#via-connection-string)?

In any event, I agree with removing this and the linearizable test from this PR. I've opened [SPEC-1315](https://jira.mongodb.org/browse/SPEC-1315) to note your suggestion about removing the corresponding `$out` tests.